.github/workflows/code-quality.yaml

```diff
@@ -0,0 +1,47 @@
+name: Code Quality
+
+on:
+    push:
+    pull_request:
+        branches: [ master ]
+    workflow_dispatch:
+
+jobs:
+    php-cs-fixer:
+        runs-on: ubuntu-latest
+        name: Coding Standards
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v3
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '8.1'
+                    coverage: none
+                    tools: php-cs-fixer, cs2pr
+
+            -   name: PHP Coding Standards Fixer
+                run: php-cs-fixer fix --dry-run --format checkstyle | cs2pr
+
+    psalm:
+        runs-on: ubuntu-latest
+        name: Static Analysis
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v3
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '8.1'
```

Comment:
@HypeMC You can target `8.2` here.

Reply:
This is intentional, let's keep the tools on a 8.1 since that's the min version the bundle supports so it makes sense to me that the check should pass on this version.

Reply:
Works for me, done